### DISCUSSION TITLE
Help needed !

### DISCUSSION THREAD

**Discussion by LionelLaffineur** [adminerevo/adminerevo#171] [General]
Hello Everyone,

I hope this message finds you well. I am reaching out to the community for assistance with our project. As this fork is getting more and more interest, there are several areas where your expertise and collaboration would be incredibly valuable. Here are the key topics where we need help:

**1. Setting Up Test Environments via Docker**

We aim to support multiple databases, and we need help setting up test environments for each one using Docker. If you have experience with Docker and database configurations, your input would be greatly appreciated. Specifically, we need:

Docker configurations for PostgreSQL, MySQL, SQLite, and other supported databases.
Scripts or documentation to easily spin up these environments for testing purposes.

**2. Expertise with Elasticsearch**

We are looking for community members who have experience with Elasticsearch. Your help is needed to:

Evaluate pull requests (PRs) and fixes related to Elasticsearch integration.
Share best practices and optimizations for using Elasticsearch within our project.
Assist with debugging and resolving any issues that arise from Elasticsearch usage.

**3. Discussing Ideas for Improvements and Roadmap**

We believe in the power of collaborative brainstorming. We encourage you to:

Share your ideas for potential improvements and new features.
Participate in discussions about the project’s roadmap and future directions.
Provide feedback on proposed changes and enhancements.

**4. Sharing the Load of Testing PRs and Double-Checking Commits**

Maintaining high-quality code is a collective effort. We need volunteers to:

Test incoming pull requests to ensure they meet our standards and do not introduce new issues.
Double-check commits for consistency, accuracy, and adherence to our coding guidelines.
Provide detailed feedback to contributors to help them improve their submissions.

**How to Get Involved ?**

If you are interested in contributing to any of the areas mentioned above, please reply to this discussion. Let us know your area of expertise and how you would like to help. Mainly ongoing participation is needed.

Looking forward to your responses and collaboration.

Best regards,

Lionel Laffineur

**Comment by RoyBellingan** [comment]
Ciao Lionel, I do normally only ever use Mariadb and Clickhouse, and I use daily adminer as is simply the best.
I have moderate xp in Php, and this is surealy an area in which I can work to improve the usage and the bug that arise.

**Reply by LionelLaffineur** [reply]
Great, I'm planning to create a branch for features for a new version and improvements that are already waiting as PRs.
Would you like to use that branch as your "daily work version" ?  (I'm also a daily user of Adminer and when I saw it was abandonned, I thought it was too sad and I created this fork ... and while working on my main job, I use the latest AdminerEvo which is the best test you can have!).
Have any other idea how you can participate ?  Any idea is welcome !

**Reply by RoyBellingan** [reply]
Oh yes is fine! And more or less same for me, I use a lot, and over time I have done a few improvement here and there, but never had confidence in releasing the code!

I have done a few suggestion, I just have to find the time to clean them!

https://github.com/adminerevo/adminerevo/discussions/202

https://github.com/adminerevo/adminerevo/discussions/89
